packages/package_info/example/android/app/src/main/AndroidManifest.xml

```diff
@@ -10,6 +10,13 @@
                   android:configChanges="orientation|keyboardHidden|keyboard|screenSize|locale|layoutDirection"
                   android:hardwareAccelerated="true"
                   android:windowSoftInputMode="adjustResize">
+        </activity>
+        <activity android:name="dev.flutter.plugins.packageinfoexample.MainActivity"
+                  android:launchMode="singleTop"
```

Comment:
FYI, I made a comment about `singleTop` in the sensor PR.

Reply:
Fixed